src/SonarQube.Client/Models/Components.cs

```diff
@@ -20,14 +20,24 @@
 
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service.DataModel
+namespace SonarQube.Client.Models
 {
-    internal class ComponentResult
+    public class ComponentDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'ComponentDTO' to match camel case naming rules, consider using 'ComponentDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)